lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1454,6 +1454,22 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     Value mult = b.create<arith::MulFOp>(loc, self, alpha);
     return b.create<arith::SubFOp>(loc, other, mult);
   }
+  if (isa<AtenToDtypeOp>(op)) {
+    Value input = payloadArgs[0];
+    Type inType = input.getType();
+    Type outType = payloadArgs[1].getType();
```

Comment:
Prefer to get the result type through `auto atenToDtype = dyn_cast<AtenToDtypeOp>(op)` and `atenToDtype.getType()` for better readability.

Reply:
We need the element type of the single element of the input tensor here. That is why I have used payLoadArgs[1].getType().

Reply:
you could use `atenToDtype.getType().cast<ValueTensorType>().getDtype()` to get the element type.

Reply:
Done. But for the input, we have to fetch from the payload args. As it needs the single entry value also below in the code.